internal/samsahai/exporter/metric.go

```diff
@@ -118,42 +119,46 @@ func SetActivePromotionMetric(activeProm *s2hv1beta1.ActivePromotion) {
 	if atpState != "" {
 		switch atpState {
 		case s2hv1beta1.ActivePromotionWaiting:
-			activePromStateList["waiting"] = 1
+			activePromStateList["waiting"] = float64(time.Now().Unix())
```

Comment:
Could we rename variable from `activePromStateList` to `atpStateList` instead to be consistence with other